src/Storage/Storage.Management/help/Get-AzStorageBlob.md

```diff
@@ -201,6 +201,20 @@ testblob             BlockBlob 2097152         application/octet-stream       20
 This command gets a single blob with blob tag condition. 
 The cmdlet will only success when the blob contains a tag with name "tag1" and value "value1", else the cmdlet will fail with error code 412.
 
+### Example 10: Get blob properties
+```powershell
+(Get-AzStorageBlob -Container "ContainerName" -Blob "blob" -Context $ctx).BlobProperties.ImmutabilityPolicy
+```
+
+```output
+ExpiresOn                   PolicyMode
+---------                   ----------
+9/17/2024 2:49:32 AM +00:00   Unlocked
+```
+
+This example command gets the immutability property of a single blob. BlobProperties cmdlet gets a detailed list of properties for a single blob.
+For a list of multiple blobs, use the cmdlet without blob name and use **ListBlobProperties** cmdlet for a better performance.
```

Comment:
The "BlobProperties" and "ListBlobProperties" are not cmdlet, but child properties of the output of "Get-AzStorageBlob" cmdlet.
```suggestion
For list multiple blobs (execute the cmdlet without blob name), use **ListBlobProperties** instead of **BlobProperties** for a better performance.
```